commands/project/open.ts

```diff
@@ -73,5 +86,12 @@ exports.builder = yargs => {
 
   yargs.example([['$0 project open', i18n(`${i18nKey}.examples.default`)]]);
 
-  return yargs;
+  return yargs as Argv<ProjectOpenArgs>;
+}
+
+module.exports = {
```

Comment:
Assuming all of these `module.exports` need to be here because the bucket hasn't been ported - could be good to add a todo to remove!

Reply:
I'm actually working on a PR now that removes these so I won't forget!